### DISCUSSION TITLE
Stop on first execution error

### DISCUSSION THREAD

**Discussion by emadzz** [webonyx/graphql-php#1166] [Q&A]
How can I force execution to stop on the first occurrence of an error?

In the documentation, it says that GraphQL is forgiving to Execution errors, however, I'd like it to be non-forgiving and stops at first error occurrence.

The current behavior results in a huge array size when these debug flags are set `DebugFlag::INCLUDE_DEBUG_MESSAGE | DebugFlag::INCLUDE_TRACE` and there are multiple entries which causes the PHP script to be exited for exceeding memory limit and it makes it difficult to debug errors.

**Comment by spawnia** [comment]
This is not possible, and I am not looking to add such an option.

**Reply by emadzz** [reply]
Is there an alternative or maybe a best practice that I can follow to avoid running into this?

**Reply by spawnia** [reply]
Make sure your server has enough memory available to handle any possible requests, including errors.
